internal/core/services/rules.go

```diff
@@ -35,7 +35,7 @@ func get(args ...interface{}) (interface{}, error) {
 		return val, nil
 	}
 
-	return nil, errNoValueFound
+	return "", nil
```

Comment:
Hmm..ok since the return type is `interface{}` let's make it return `nil, nil` instead.